src/app/sections/ProbeDetails.svelte

```diff
@@ -99,34 +89,53 @@ h2 {
     margin-bottom: var(--space-base);
 }
 
-.details-list {
+.info-type {
     margin: 0;
-    padding: 0;
-    list-style: none;
 }
 
-.details-list > * {
-    margin: 0 0 var(--space-base) 0;
+.info-type > * {
+    line-height: 1.2;
 }
 
-.details-list > *:last-child {
-    margin-bottom: 0;
+.info-type dt {
+    text-transform: capitalize;
+    color: var(--subhead-gray-01);
+    font-size: var(--text-04);
+    font-weight: 500;
 }
 
-.detail--indented {
-    position: relative;
-    padding-left: var(--space-2x);
+.info-type dd {
+    margin: 0;
     font-size: var(--text-015);
+    color: var(--subhead-gray-01);
 }
 
-.detail--indented::before {
-    content: ">";
-    position: absolute;
-    height: var(--space-base);
-    width: var(--space-base);
-    top: 0;
-    left: 0;
-    color: var(--cool-gray-400);
+.info-type--padded {
+    padding: 0 9px;
+}
+
+.info-type--subtle {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 0;
```

Comment:
```suggestion
    padding-top: 0;
    align-items: baseline;
```

Reply:
the `align-items: baseline` will make sure the vertical alignment matches b/t the number and the channel. By default I think it is `start` which looks off